src/app/interceptors/error-handling.interceptor.spec.ts

```diff
@@ -254,4 +258,55 @@ describe('ErrorHandlingInterceptor', () => {
         expect(refreshSpy).toHaveBeenCalledTimes(1);
         expect(notificationService.apiError).toBeFalsy();
     });
+
+    it('should handle non-response Raiden API errors for path url requests', () => {
+        raidenConfig.api = '/api/v1';
+        // Unlike the HttpClientModule, the HttpClientTestingModule does not prepend the url
+        // property of a HttpResponse with the host if the request was made by a URL path.
+        const requestUrl = window.location.origin + raidenConfig.api;
+        const errorSpy = jasmine.createSpy();
+
+        service.makeRequest(requestUrl).subscribe(() => {
+            fail('On next should not be called');
+        }, errorSpy);
+
+        const request = httpMock.expectOne(requestUrl);
+        request.flush(
+            {},
+            {
+                status: 504,
+                statusText: '',
+            }
+        );
+
+        expect(notificationService.addErrorNotification).toHaveBeenCalledTimes(
+            1
+        );
+        expect(errorSpy).toHaveBeenCalledTimes(1);
+    });
+
+    it('should not misinterpret non-api errors', () => {
+        raidenConfig.api = '/api/v1';
+        const requestUrl = 'http://some.url/api/v1';
+        const errorSpy = jasmine.createSpy();
+
+        service.makeRequest(requestUrl).subscribe(() => {
+            fail('On next should not be called');
+        }, errorSpy);
+
+        const request = httpMock.expectOne(requestUrl);
+        request.flush(
+            {},
+            {
+                status: 504,
+                statusText: '',
+            }
```

Comment:
Nitpick: You seem to repeat these 4 lines over and over again. Maybe it saves you lines, times and readability if you make it a constant like `http_response_504` or so? It would make 7 lines become 1 at many places. Just an idea. 🤷🏾